data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -2248,10 +2248,24 @@ spec:
                       specifies which project the network and subnets exist in when
                       they are not in the main ProjectID.
                     type: string
+                  privateManagedZone:
+                    description: PrivateManagedZone is currently unsupported. The PrivateManagedZone should be
+                      provided as "projects/{project-id}/managedZones/{managed-zone-name}". When the PrivateManagedZone
+                      is empty, the DNS zones will be created and managed in the Service Project (ProjectID). When
+                      "None", the DNS zones will be externally managed. When PrivateManagedZone is provided, the DNS
```

Comment:
Can we make "None" as "External" ? I feel that the semantics of an empty string and "None" are quite different but the values could be conflated easily while providing configuration. Using "external" makes the intent specific.